### PR TITLE
Add support for glob env files

### LINKED ISSUE #214: Add support for glob pattern for envFiles
It would be beneficial to read all environment files from a glob pattern. This would be helpful in cases where the envfiles are generated and removed on the fly based on some conditions. The qbec.yaml would not need to be updated every time a new envfile is added or created.

### PR DESCRIPTION
**Pull request by harsimranmaan**
Glob patterns can be supplied for envFiles in qbec.yaml as well as from the command-line. The pattern must match at-least a file on disk. http env files are not evaluated for glob patterns.

Closes #214

### CODE REVIEW

File: internal/filematcher/match.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	}
+	var envFiles []string
+	for _, f := range files {
+		abs, err := filepath.Abs(f)
+		if err != nil {
+			return nil, err
+		}
+		envFiles = append(envFiles, abs)
+	}
+	return envFiles, nil
```
Comment (gotwarlost) [inline internal/filematcher/match.go:31]: needs a sort because `Glob` does not guarantee this and we always need a predictable order

Comment (gotwarlost) [reply internal/filematcher/match.go:31]: also update the function comment to reflect this

**Comment by gotwarlost** [comment]
and codecov doesn't deliver for the PRs that you really need it :)

**Comment by gotwarlost** [comment]
apart from the missing sort, LGTM

**Comment by harsimranmaan** [comment]
> and codecov doesn't deliver for the PRs that you really need it :)

I read on a community forum that the bot might be hitting rate limits for posting comments on the PRs.

**Pull request merged**